config/environments/test.rb

```diff
@@ -0,0 +1,42 @@
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [85/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)